plutus-tx-plugin/src/PlutusTx/Plugin.hs

```diff
@@ -390,6 +390,7 @@ compileMarkedExpr locStr codeTy origE = do
           , 'useToOpaque
           , 'useFromOpaque
           , 'mkNilOpaque
+          , 'PlutusTx.Builtins.equalsInteger
```

Comment:
Will GHC get pick over `PlutusTx.Builtins.equalsInteger` vs `Builtins.equalsInteger`? Initially, I had `Builtins.equalsInteger` in `builtinNames` but that didn't provided `equalsInteger` in the context and I got `PlutusTx.Builtins.equalsInteger` not found error from plugin; hence I went through the rabbit hole.